src/components/Navbar/Hamburger.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import Link from 'next/link'
+import NavbarAccount from './Account'
+import { useRouter } from 'next/router'
+
+const Hamburger = () => {
+  const router = useRouter()
+
+  const handleProductClick = () => {
```

Comment:
Since this function is only for handling navigation. Should we use `Link` from `next/link` here instead?